moviepy/video/io/ffmpeg_writer.py

```diff
@@ -221,13 +221,12 @@ def ffmpeg_write_video(
     codec="libx264",
     bitrate=None,
     preset="medium",
-    with_mask=False,
```

Comment:
`with mask` when set would only set the `pixel_format` to have alpha.
I think we can remove it and default to "rgb24". If anyone wants the `pixel_format` to have the alpha layer, they can pass it through the pixel_format itself.